src/CompanyService.Business/Commands/ContractSubject/CreateContractSubjectCommand.cs

```diff
@@ -44,8 +44,7 @@ public class CreateContractSubjectCommand : ICreateContractSubjectCommand
 
     public async Task<OperationResultResponse<Guid?>> ExecuteAsync(CreateContractSubjectRequest request)
     {
-      if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveCompanyData)
-        || !await _accessValidator.HasRightsAsync(Rights.AddEditRemoveCompanies))
+      if (!await _accessValidator.HasRightsAsync(Rights.AddEditRemoveCompanyData, Rights.AddEditRemoveCompanies))
```

Comment:
why both? not any?

Reply:
Old logic: if one of rights is absent return false so it means that we need both

Reply:
stay only CompanyData here